charts/alfresco-audit-storage/templates/deployment.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "alfresco-audit-storage.fullname" . }}
+  labels:
+    {{- include "alfresco-audit-storage.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "alfresco-audit-storage.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        checksum/config-es: {{ include (print $.Template.BasePath "/configmap-es.yaml") $ | sha256sum }}
+        checksum/config-mq: {{ include (print $.Template.BasePath "/configmap-mq.yaml") $ | sha256sum }}
+        checksum/secret-es: {{ include (print $.Template.BasePath "/secret-es.yaml") $ | sha256sum }}
+        checksum/secret-mq: {{ include (print $.Template.BasePath "/secret-mq.yaml") $ | sha256sum }}
```

Comment:
I would not use that as it do not fit with the approach we use widely to offer usage of exisitingConfigmap (and has other drawbacks). We've been removing it from other charts (although some are probably still around). I think we should have a consistent recommendation of using proper tools like [reloader](https://github.com/stakater/Reloader)

Reply:
I think we never took a final decision on this, for me they are still worth to have because **sometimes** they help the end users without requiring any external tooling, so dropping them mean making life harder **every time** for end users.

Reply:
service restarts should not really be an issue on a kubernetes environment, plus this approach is also documented on helm docs https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments

however not a big deal for me if you prefer to get rid of it